lessons/pydata/clt/index.ipynb

```diff
@@ -0,0 +1,690 @@
+{
```

Comment:
<p>Přípustná chyba čeho? Pravděpodobně myslíš chybu zastoupení něčeho v celku, asi by stálo za to to explicitně uvést.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>